src/main/resources/application.properties

```diff
@@ -34,3 +34,20 @@ booking.ics.myCal.directory=src/main/resources/calendar/my-cal
 # cors
 cors.allowedOrigins=http://localhost:3000
 
+# Email Confirmation URL
+confirmation.url=http://localhost:8082/api/bookings/resend-confirmation/
```

Comment:
this probably should be ${BASE_URL}/api/booking/resend-confirmation
Because when we deploy the project the BASE_URL is not localhost.